app.go

```diff
@@ -143,6 +145,14 @@ func (a *App) Setup() {
 		if c.HelpName == "" {
 			c.HelpName = fmt.Sprintf("%s %s", a.HelpName, c.Name)
 		}
+
+		fc := FlagCategories{}
+		for _, flag := range c.Flags {
+			fc = fc.AddFlag(flag.GetCategory(), flag)
+		}
+
+		sort.Sort(fc)
```

Comment:
Hello,

It is possible to add an option to be able to disable default sorting? I would like to sort categories by appearance :

```bash
...
OPTIONS:
  scope
   --scope1 value     object_code
   --scope2 value   object test

  mandatory
   --john value  JJ
   --doe value       DD
```

```
Flags: []cli.Flag{
			cli.StringFlag{
				Name:  "scope1",
				Category: "scope",
			},
			cli.StringFlag{
				Name:  "scope2",
				Category: "scope",
			},
			cli.StringFlag{
				Name:  "john",
				Category: "mandatory",
			},
			cli.StringFlag{
				Name:  "doe",
				Category: "mandatory",
			},
```

Thanks !